src/main/java/com/edu/BaseballgameMain.java

```diff
@@ -0,0 +1,40 @@
+package com.edu;
+
+import java.util.Scanner;
+
+public class BaseballgameMain {
+
+    public static void main(String[] args) {
+
+        Scanner scan = new Scanner(System.in);
```

Comment:
```suggestion
        Scanner scan = new Scanner(System.in);
```
자바 코드 컨벤션에 의하면 메소드의 시작부분은 공백없이 붙여서 사용합니다.